### ISSUE TITLE
How to listen to the audio

### ISSUE THREAD

**Issue by NalaBH** [markondej/fm_transmitter#94]
I set everything up and I run the program in MU and it says the audio is playing but I dont hear anything

**Comment by Enderiscool** [comment]
you use fm radio? like in a car

**Comment by WyeHacker** [comment]
yes, this project runs through FM radio, tune in the frequency and listen on a Radio
